.github/workflows/lint-python.yml

```diff
@@ -22,7 +22,10 @@ jobs:
       uses: actions/checkout@v4
 
     - name: Install yapf
-      run: pip install yapf==0.40.1
+      run: |
+        python3 -mvenv /tmp/venv/yapf
```

Comment:
fyi instead of manually creating a `venv`, could also use the [setup-python](https://github.com/actions/setup-python) action which updates the `PATH` with isolated python interpreter and save from having to `source` the `activate` script, e.g.:
```yaml
      - uses: actions/setup-python@v5
        with:
          python-version: '3.10'
```